input/docs/cs/handwritten_to_digital.md

```diff
@@ -0,0 +1,19 @@
+title: Fotka s ručně psanými čísly na binární obraz
+Description: Konverze ručně psaných čísel do binárního obrazu
+---
+>Tento dokument pracuje s [obrázkem čísel na papíře](/zodoc/assets/img/cisla_ver.jpg) uložený v proměnné `A`.
```

Comment:
Obrázek už tam je v zodoc/média, je to vhodnější umístění než v assets, kde je momentálně kytka. Omlouvám se za zmatky. Tak v assets obrázek smaž a odkaž na media..